pkg/controllers/oauthendpoints/oauth_endpoints_controller.go

```diff
@@ -65,7 +77,9 @@ func NewOAuthRouteCheckController(
 			routeInformer,
 			ingressInformer,
```

Comment:
Good point, I believe we should be watching the ingress; it was not added to the filter so it would have been ignored, so I've added it to the filter :+1: